### ISSUE TITLE
Use <= and < instead of > or >= in comparison

### ISSUE THREAD

**Issue by casper91** [rubocop-hq/rubocop#6523]
Hi. 
I have tried to search in the docs and on the web for a Rubocop rule to use <= and < instead of > or >= in comparison, but can't seem to find anything about it. 

Direction is unordered. However the direction should be consistent across the base. 

Example: "ascending"
```Ruby
# bad
a > 2
3 >= b
2 <= c && 4 > c

#good
2 < a
b <= 3
4 < c && c <= 7
```

Example: "descending"
```Ruby
# bad
2 < a
b <= 3
2 <= c && 4 > c

#good
a > 2
3 >= b
7 >= c && c > 4
```

It can conflict with `YodaCondition` however I find it easier/faster to figure out when only using the same operators.

**Comment by ekampp** [comment]
I'm interested in why you believe comparison in one direction is better than the other, or is the issue simply that which ever way the comparison goes it should be consistent across the code base?

Consider giving value objects a look for this sort of work, since that can encapsulate simple type comparison pretty well ')

**Comment by mikegee** [comment]
This seems like a reasonable thing for Rubocop to support. I think ascending comparisons are more natural coming from a mathematical background.

(@casper91 I think the first two "bad" "descending" examples are already descending.)

**Comment by casper91** [comment]
Yeah, "ascending" is also my preferred choice.
Updated Example, thanks!

**Comment by ekampp** [comment]
I find that `a > 2` is more clear than `2 < a`, because the variable is the first thing I read, so cognitively it takes up fewer resources. `<variable> <operator> <value>` should perhaps be the default.

**Comment by mikegee** [comment]
@ekampp, Rubocop currently enforces that style by default with [YodaCondition](https://rubocop.readthedocs.io/en/latest/cops_style/#styleyodacondition). This issue is a feature request for another option.

**Comment by ekampp** [comment]
Great, then don't break that with this feature by enforcing this feature in a specific direction.

**Comment by mikegee** [comment]
Right on. This feature would need to be another option folks could opt into.
